FalmouthFalcons/iteration-3
# Ticket9/jq

Description:
-User can delete a product if they created it. Product is set to false and remove from system once its no longer available.

Number of Fixes:
-one

Related Ticket(s):
- Ticket 9

Problem to Solve:
- User can delete a product if they created it. Product is set to false and remove from system once its no longer available.

Proposed Changes:
-User can delete a product by user id

Expected Behavior:
-User can delete a product by user id

Steps to Test Solution
1) git fetch
2) git pull ticket9/jq
3) products page, click destroy product should change active to false and update DB

Documentation
[ ] There is new documentation in this pull request that must be reviewed..
[ ] I added documentation for any new classes/methods

Deploy Notes:
- Notes regarding deployment the contained body of work.
- These should note any db migrations, etc.
